GitUI/CommandsDialogs/RevisionFileTreeControl.cs

```diff
@@ -51,26 +51,29 @@ public partial class RevisionFileTreeControl : GitModuleControl
         private readonly Stack<string> _lastSelectedNodes = new();
         private readonly IRevisionFileTreeController _revisionFileTreeController;
         private readonly IFullPathResolver _fullPathResolver;
-        private readonly IFindFilePredicateProvider _findFilePredicateProvider;
+        private readonly IFindFilePredicateProvider _findFilePredicateProvider = new FindFilePredicateProvider();
         private GitRevision? _revision;
         private readonly RememberFileContextMenuController _rememberFileContextMenuController
             = RememberFileContextMenuController.Default;
         private Action? _refreshGitStatus;
+        private RevisionGridControl? _revisionGrid;
 
         public RevisionFileTreeControl()
         {
             InitializeComponent();
             InitializeComplete();
             HotkeysEnabled = true;
             _fullPathResolver = new FullPathResolver(() => Module.WorkingDir);
-            _findFilePredicateProvider = new FindFilePredicateProvider();
             _revisionFileTreeController = new RevisionFileTreeController(() => Module.WorkingDir,
                                                                          new GitRevisionInfoProvider(() => Module),
                                                                          new FileAssociatedIconProvider());
+            BlameControl.HideCommitInfo();
+            blameToolStripMenuItem1.Checked = AppSettings.RevisionFileTreeShowBlame;
         }
 
-        public void Bind(Action refreshGitStatus)
+        public void Bind(RevisionGridControl? revisionGrid, Action refreshGitStatus)
         {
+            _revisionGrid = revisionGrid;
             _refreshGitStatus = refreshGitStatus;
```

Comment:
Since you're using nullable reference type for `RevisionGridControl`, I'm wondering if the `Action refreshGitStatus `parameter should be too `Action? refreshGitStatus` here?

Reply:
That depends on what the caller provides, but in this case it should be changed
(comments on existing code is welcome, bigger changes should be separated commits or even PRs though)

Reply:
`RevisionGridControl` should not be NRT, caller is always non null
In RevDiff `refreshGitStatus` should be NRT too.